src/index.js

```diff
@@ -1,20 +1,28 @@
 'use strict';
 
 import assign from 'lodash.assign';
+import path from 'path';
+import url from 'url';
+
 import ResultList from './result-list';
 
 const requiredDefaults = {
   page: 1,
   per_page: 20,
+  url: 'https://pixabay.com/api'
 };
 
 const pixabayjs = {
   _auth: {},
 
-  _makeConfig: function(url, opts) {
-    const urlObj = { url };
-    return assign({}, requiredDefaults, urlObj, this.defaults, this._auth,
-      opts);
+  _makeConfig: function(endpoint, opts) {
+    const conf = assign({}, requiredDefaults, this.defaults, this._auth, opts);
+    const urlObj = url.parse(conf.url);
+
+    urlObj.pathname = path.join(urlObj.pathname, endpoint, '/');
```

Comment:
It's not required in the sense that the url will still resolve correctly, but it is better to include the trailing slash.

https://cdivilly.wordpress.com/2014/03/11/why-trailing-slashes-on-uris-are-important/